app/src/main/java/com/eveningoutpost/dexdrip/utils/LocationHelper.java

```diff
@@ -0,0 +1,71 @@
+package com.eveningoutpost.dexdrip.utils;
+
+import android.app.Activity;
+import android.app.AlertDialog;
+import android.content.Context;
+import android.content.DialogInterface;
+import android.content.Intent;
+import android.location.LocationManager;
+import android.os.Build;
+
+import com.eveningoutpost.dexdrip.R;
+
+/**
+ * Helper for checking if location services are enabled on the device.
+ */
+public class LocationHelper {
+    // Manually defining Marshmallow to maintain backwards API compatibility.
+    // http://developer.android.com/reference/android/os/Build.VERSION_CODES.html#M
+    private static final int MARSHMALLOW = 23;
```

Comment:
Why can't `Build.VERSION_CODES.M` be used?
This should be set at compile time and not at runtime.